src/utils/convert.js

```diff
@@ -170,7 +170,7 @@ const renderStep = (type, definition, key) => {
   const templateContext = {
     KEY: snakeCase(key),
     CAMEL: camelCase(key),
-    NOUN: _.capitalize(key),
+    NOUN: definition.noun,
     LOWER_NOUN: key.toLowerCase(),
```

Comment:
Is this used? If so, might want to apply a similar logic as above.

Reply:
Also fixed in 53cd6d3. Thanks :)